content/en/docs/concepts/workloads/pods/downward-api.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Downward API
+content_type: concept
+description: >
+  There are two ways to expose Pod and container fields to a running container:
+  environment variables, and as files that are populated by a special volume type.
+  Together, these two ways of exposing Pod and container fields are called the downward API.
+---
+
+<!-- overview -->
+
+It is sometimes useful for a container to have information about itself, without
+being overly coupled to Kubernetes. The _downward API_ allows containers to consume
+information about themselves or the cluster without using the Kubernetes client
+or API server.
+
+An example is an existing application that assumes a particular well-known
+environment variable holds a unique identifier. One possibility is to wrap the
+application, but that is tedious and error prone, and it violates the goal of low
```

Comment:
Normally I am treasurer for life of the I Hate Hyphens club, but for the sake of easier localization--
```suggestion
application, but that is tedious and error-prone, and it violates the goal of low
```